### PR TITLE
Add Unsplash attribution at the bottom left

### PR DESCRIPTION
**Pull request by diegovar**
## Summary

Adds the Unsplash attribution as per the mocks at the bottom left, when the provider type is Unsplash.

Also, this fixes an alignment issue that the gallery had caused by the margins around the media elements and the category pills.

Screen Shot 2020-07-30 at 6 08 55 pm

## Relevant Technical Choices

Had to get the SVG from the Unsplash website. This approach is not really internationalizable, we'd need to either get proper images for each languages, or use some sort of i18n interpolation.

## To-do

Hide the attribution when the loading pill is visible.

## User-facing changes

The Unsplash attribution is now visible.

## Testing Instructions

Enable the media3pTab flag.

---

Addresses #3357

### CODE REVIEW

File: assets/src/edit-story/components/library/panes/media/common/paginatedMediaGallery.js [hunk 1]
```diff
@@ -36,16 +36,39 @@ import { __ } from '@wordpress/i18n';
...
+  line-height: 16px;
+  display: flex;
+  flex-wrap: nowrap;
+  font-size: 12px;
+  background-color: rgba(0, 0, 0, 0.7);
+  cursor: pointer;
+`;
+
+const LOGO_PROPS = {
+  fill: '#fff',
```
Comment (swissspidy) [inline assets/src/edit-story/components/library/panes/media/common/paginatedMediaGallery.js]: Theme?

Comment (diegovar) [reply assets/src/edit-story/components/library/panes/media/common/paginatedMediaGallery.js]: Done

File: assets/src/edit-story/components/library/panes/media/common/paginatedMediaGallery.js [hunk 2]
```diff
@@ -36,16 +36,40 @@ import { __ } from '@wordpress/i18n';
...
+const AttributionPill = styled.div`
+  position: absolute;
+  left: 24px;
+  bottom: 10px;
+  border-radius: 100px;
+  padding: 5px 8px;
+  line-height: 16px;
+  display: flex;
+  flex-wrap: nowrap;
+  font-size: 12px;
```
Comment (winterDroid) [inline assets/src/edit-story/components/library/panes/media/common/paginatedMediaGallery.js:63]: Shall the font size, colors and line-height come from the theme?

File: assets/src/edit-story/components/library/panes/media/common/paginatedMediaGallery.js [hunk 3]
```diff
@@ -141,6 +165,12 @@ function PaginatedMediaGallery({
     };
   }, [handleScrollOrResize]);
 
+  const openLink = useCallback(() => {
+    if (providerType === ProviderType.UNSPLASH) {
+      window.open('https://unsplash.com');
```
Comment (winterDroid) [inline assets/src/edit-story/components/library/panes/media/common/paginatedMediaGallery.js]: Shall this open in a new tab / window?

**Review by winterDroid — APPROVED** [review]

File: assets/src/edit-story/components/library/panes/media/common/paginatedMediaGallery.js (hunk 3 above)
Comment (swissspidy) [reply assets/src/edit-story/components/library/panes/media/common/paginatedMediaGallery.js]: Why window.open and not just a hyperlink? Would be better for accessibility for sure.

Comment (diegovar) [reply assets/src/edit-story/components/library/panes/media/common/paginatedMediaGallery.js]: Good point, updated.

File: assets/src/edit-story/components/library/panes/media/common/paginatedMediaGallery.js (hunk 2 above)
Comment (diegovar) [reply assets/src/edit-story/components/library/panes/media/common/paginatedMediaGallery.js:63]: I see the advantage of using the theme for the background-color and color, but I question what the value is for the font size, line-height, etc. The theme should provide semantically charged configurations that allow us to switch eventually between themes (dark and light mode). In this case what would be the advantage to create a new attribution font size and line height in the theme when it won't be used anywhere else in the application?

Comment (winterDroid) [reply assets/src/edit-story/components/library/panes/media/common/paginatedMediaGallery.js:63]: I also see the biggest value in the colors. But other places also make use of it for sizes. I think it makes it easier to be consistent and see all resources in one place. It is done e.g. similarly in Android. But I'm not strong about it.

Comment (diegovar) [reply assets/src/edit-story/components/library/panes/media/common/paginatedMediaGallery.js:63]: For example, there is no rgba(0,0,0,0.7) in the theme. Am I supposed to create a new 
```
media3p: {
  attribution: {
    bg: rgba(0,0,0,0.7)
  }
}
```
in there, or simply add a new bg color like
```
colors: {
  bg: {
    v16: rgba(0,0,0,0.7)
  }
}
```
It's very unclear how themes are meant to be used IMO.

Comment (diegovar) [reply assets/src/edit-story/components/library/panes/media/common/paginatedMediaGallery.js:63]: In this case the font-size and line-height has been fine tuned to match the height of the unsplash SVG logo's font, so I'd rather leave them as hardcoded values in the css.

Comment (swissspidy) [reply assets/src/edit-story/components/library/panes/media/common/paginatedMediaGallery.js:63]: `media3p.attribution` would have been fine.

**Pull request merged**